simpleLooping/removeSpaceAtEnd.py

```diff
@@ -17,3 +17,13 @@
 #   removeSpaceAtEnd("")                  # return ""
 #   removeSpaceAtEnd("Hello  World!")     # return "Hello  World!"
 #   removeSpaceAtEnd("  Hello  World!  ") # return "  Hello  World!"
+from typing import List
+
+def removeSpaceAtEnd(s: str) -> str:
+    lenLst = len(s)
+
+    i = lenLst - 1
```

Comment:
Is it worth it to have those 2 lines separately like this?

lenLst = len(s)

i = lenLst - 1

Reply:
acknowledge.